contracts/src/GMXSampler.sol

```diff
@@ -21,13 +21,22 @@ contract GMXSampler is SamplerUtils, ApproximateBuys {
         uint256 numSamples = takerTokenAmounts.length;
         makerTokenAmounts = new uint256[](numSamples);
         for (uint256 i = 0; i < numSamples; i++) {
-            try IGMX(reader).getAmountOut(IVault(vault), path[0], path[1], takerTokenAmounts[i]) returns (
```

Comment:
nit/optional: this would make it cleaner.
```
    function sampleSellsFromGMX(
        IGMX reader,
        IVault vault,
        address[] memory path,
        uint256[] memory takerTokenAmounts
    )
 ```